### PR TITLE
Add BSD license explicitly

### PR DESCRIPTION
**Pull request by hwware**
Add "BSD 3-Clause License" in License 1 and License 2 part

### CODE REVIEW

**Comment by madolson** [comment]
Is there a specific reason you want this? I'm okay with adding this clarification, just not exactly sure what the intention is.

**Comment by hwware** [comment]
Before we deliver the software, we need to check the specific software license. But now we miss the explicit words in source codes, thus we wish to add this.

File: COPYING [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# License 1
+# License 1 (The 3-Clause BSD License)
```
Comment (madolson) [inline COPYING]:
```suggestion
# License 1

BSD 3-Clause License
```

I see this is how it's written by default when asking github to do it. Maybe someone else is looking for it on its own line. I suppose we should also update License 2 in this file to say the same thing? Are you specifically looking for the wording you provided?

**Comment by zuiderkwast** [comment]
I agree that it's good to mention the name of the license. Not everyone can recognize it from the license text.

File: COPYING (hunk 1 above)
Comment (hwware) [reply COPYING]: I think adding BSD 3-Clause License words in License 2 part is fine as well.  I think there is no other specific words. Thanks

**Review by madolson — APPROVED** [review]

**Pull request merged**